module/Company/view/company/company/job-list.phtml

```diff
@@ -59,6 +59,10 @@ $this->headTitle($this->translate('Jobs'));
         <?php
 
 
+        // Randomize the order of companies
+        if ($randomize):
+            shuffle($jobList);
+        endif
```

Comment:
Within view code, we have two styles of using control statements like `if`:

```php
<?php if (/* condition*/): ?>
HTML
<?php endif ?>
```

And:

```php
<?php
if (/* condition */) {
    /* PHP code */
}
```

Mixing them only makes code less readable.